app/api/apiConfig.js

```diff
@@ -1,6 +1,5 @@
 export const blockTradesAPIs = {
     BASE: "https://api.blocktrades.us/v2",
-    // BASE_OL: "https://api.blocktrades.us/ol/v2",
     BASE_OL: "https://ol-api1.openledger.info/api/v0/ol/support",
```

Comment:
It doesn't really make sense to reuse this key as `BASE_OL` (base openledger) and duplicating values for the other APIs just to map to the same key. Now that Openledger handles their own gateway service it would make more sense to create a new `const openledgerAPIs`, and use the `BASE` key for all the different APIs.

Reply:
My plan was to make this transition in the next issue I made.  I didn't want to dig to deep just to make Winex work. Unless you disagree, and I could do this here?